makefiles/pojavlauncher.mk

```diff
@@ -33,10 +33,12 @@ pojavlauncher: pojavlauncher-setup
 			mkdir -p build; \
 			cd build; \
 			wget https://github.com/leetal/ios-cmake/raw/master/ios.toolchain.cmake; \
-			cmake .. -G Xcode -DDEPLOYMENT_TARGET=$(IPHONEOS_DEPLOYMENT_TARGET) -DCMAKE_TOOLCHAIN_FILE=ios.toolchain.cmake -DPLATFORM=OS64 -DENABLE_BITCODE=FALSE -DCMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_REQUIRED="NO" -DCMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_ALLOWED=NO -DCMAKE_XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY=""; \
+			cmake .. -G Xcode -DCMAKE_TOOLCHAIN_FILE=ios.toolchain.cmake -DDEPLOYMENT_TARGET=$(IPHONEOS_DEPLOYMENT_TARGET) -DENABLE_ARC=TRUE -DENABLE_VISIBILITY=FALSE -DPLATFORM=OS64 -DENABLE_BITCODE=FALSE -DENABLE_STRICT_TRY_COMPILE=FALSE -DCMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_REQUIRED="NO" -DCMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_ALLOWED=NO -DCMAKE_XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY=""; \
 			cmake --build . --config Release --target pojavexec PojavLauncher; \
 			cd ../..; \
 			mkdir -p Natives/build/Release-iphoneos/PojavLauncher.app/Base.lproj; \
+			/Applications/Xcode.app/Contents/Developer/usr/bin/actool Natives/Assets.xcassets --compile Natives/resources --platform iphoneos --minimum-deployment-target $(IPHONEOS_DEPLOYMENT_TARGET) --app-icon AppIcon --output-partial-info-plist /dev/null; \
```

Comment:
Can you change this to just “xcrun actool”

Reply:
If I had the fr emote available here, I would’ve used it. In my other pull that vanished, you said to use the direct path.

Reply:
![fr](https://cdn.discordapp.com/emojis/792934328236638267.png?v=1)

Reply:
fr chain
![fr](https://camo.githubusercontent.com/0f1480805d876b2222c4ae48d8b5781bbd379e770df6e4a7d2094963b6b6513e/68747470733a2f2f63646e2e646973636f72646170702e636f6d2f656d6f6a69732f3739323933343332383233363633383236372e706e673f763d31)

Reply:
!fr

Reply:
oof

Reply:
![fr](https://camo.githubusercontent.com/0f1480805d876b2222c4ae48d8b5781bbd379e770df6e4a7d2094963b6b6513e/68747470733a2f2f63646e2e646973636f72646170702e636f6d2f656d6f6a69732f3739323933343332383233363633383236372e706e673f763d31)